client/src/preview/route/digitaltwins/manage/DeleteDialog.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import { Dispatch, SetStateAction } from 'react';
+import {
+  Dialog,
+  DialogContent,
+  DialogActions,
+  Button,
+  Typography,
+} from '@mui/material';
+import { useDispatch, useSelector } from 'react-redux';
+import { selectDigitalTwinByName } from '../../../store/digitalTwin.slice';
+import DigitalTwin, { formatName } from '../../../util/gitlabDigitalTwin';
+import { showSnackbar } from '../../../store/snackbar.slice';
+
+interface DeleteDialogProps {
+  showLog: boolean;
+  setShowLog: Dispatch<SetStateAction<boolean>>;
+  name: string;
+  onDelete: () => void;
+}
+
+const handleCloseLog = (setShowLog: Dispatch<SetStateAction<boolean>>) => {
+  setShowLog(false);
+};
+
+const handleDelete = async (
+  digitalTwin: DigitalTwin,
+  setShowLog: Dispatch<SetStateAction<boolean>>,
+  onDelete: () => void,
```

Comment:
`onDelete` function is void everywhere in the codebase. Is it needed?

Reply:
Yes, it does not have a return value but modifies the store via dispatch.